R/sampler.R

```diff
@@ -49,28 +51,25 @@ particle_filter <- function(data, model, compare, n_particles,
   }
 
   i_state <- seq_along(model$initial()) + 1L
-
-  ## Special treatment for the burn-in phase; later we might use this
-  ## same approach for skipping steps though.
-  if (save_particles) {
+  
+  if (save_particles){
```

Comment:
Remove this section (see comments below)

Reply:
Done